packages/compass-sidebar/src/components/legacy/navigation-items.spec.tsx

```diff
@@ -9,6 +9,8 @@ import reducer from '../../modules';
 import { NavigationItems } from './navigation-items';
 import { WorkspacesProvider } from '@mongodb-js/compass-workspaces';
 
+const CONNECTION_ID = 'webscale';
```

Comment:
🤣

Reply:
I am gonna have to watch this again, this triggered me 🤣

Reply:
![image](https://github.com/mongodb-js/compass/assets/1822138/4a32cf06-128b-4ceb-a334-53223d44b0a7)

Reply:
oh wow, I haven't seen this before and now I'm dying 😂